src/core/tests/Test_CoreMain.py

```diff
@@ -73,6 +74,7 @@ def test_operation_fail_for_autopatching_request(self):
         argument_composer.maintenance_run_id = str(datetime.datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%S.%fZ"))
         runtime = RuntimeCompositor(argument_composer.get_composed_arguments(), True, Constants.ZYPPER)
         runtime.set_legacy_test_type('FailInstallPath')
+        time.sleep = runtime.mock_sleep
```

Comment:
Isn't time.sleep already set with mocked code in RuntimeCompositor. See under the comment 'Overriding time.sleep' and urlopen...' in RuntimeCompositor __init_()

Reply:
Dead code from trying something else -- good catch